eZ/Publish/SPI/Repository/Tests/Decorator/URLWildcardServiceDecoratorTest.php

```diff
@@ -35,7 +35,7 @@ public function testCreateDecorator()
         $parameters = [
             'random_value_5ced05ce0f76d9.50834569',
             'random_value_5ced05ce0f7709.88573575',
```

Comment:
It would be great to improve these values at once, something like that
```
        'source_url_value',
        'destination_url_value',
```
It could improve code readability

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2855/commits/76d00d7be428e7e8c650c56cf9b33d139bc909ac